scripts/deploy-commands.js

```diff
@@ -2,30 +2,44 @@
 // Commands are deployed globally by default.
 // This means that it may take a little bit of time before your commands can be seen on all guilds.
 
-const fs = require('fs'), path = require('path');
-const { REST } = require('@discordjs/rest');
-const { Routes } = require('discord-api-types/v9');
-const { applicationId, token } = require('#settings');
+import { readdirSync } from 'fs';
+import { REST } from '@discordjs/rest';
+import { Routes } from 'discord-api-types/v9';
+import { Collection } from 'discord.js';
+import { token, applicationId } from '#settings';
+import { getAbsoluteFileURL } from '#lib/util/util.js';
+import { setLocales } from '#lib/util/common.js';
+
+const locales = new Collection();
+const localeFolders = readdirSync(getAbsoluteFileURL(import.meta.url, ['..', 'locales']));
+for await (const folder of localeFolders) {
```

Comment:
I think the `await` here only matters for the second `for...of` loop inside this.
```suggestion
for (const folder of localeFolders) {
```

Reply:
couldn't find anything that proves if a for await is required for the for await within to work properly, but I'll leave it for now since ESLint isn't complaining either way